### ISSUE TITLE
Minimal Theme Settings plugin causing issue with note padding

### ISSUE THREAD

**Issue by adamhl8** [kepano/obsidian-minimal#616]
### Is the bug present when using the default Obsidian theme?

No

### Is the bug present when snippets and plugins are disabled?

No

### Minimal theme version

7.0.7

### Describe the bug

When I have the Minimal Theme Settings plugin enabled, it seems like I lose some padding inside my notes.

Enabled:
![image](https://github.com/kepano/obsidian-minimal/assets/1844269/cda3bed8-fb71-4073-b6da-eed9f47ae9a0)

Disabled:
![image](https://github.com/kepano/obsidian-minimal/assets/1844269/e2e27a03-252e-4268-84c4-7abb8a08662d)

Even after turning all features off in the plugin settings (and restarting Obsidian) the issue persists. Only disabling the plugin completely makes the issue go away.

### Debug info

```
SYSTEM INFO:
	Obsidian version: v1.3.7
	Installer version: v1.3.5
	Operating system: Darwin Kernel Version 22.5.0: Thu Jun  8 22:22:23 PDT 2023; root:xnu-8796.121.3~7/RELEASE_ARM64_T6020 22.5.0
	Login status: logged in
	Catalyst license: insider
	Insider build toggle: on
	Live preview: off
	Legacy editor: off
	Base theme: dark
	Community theme: Minimal v7.0.7
	Snippets enabled: 2
	Restricted mode: off
	Plugins installed: 17
	Plugins enabled: 17
		1: Emoji Shortcodes v2.2.0
		2: Homepage v3.3.2
		3: Regex Find/Replace v1.2.0
		4: Tasks v4.3.0
		5: Icon Folder v2.1.2
		6: Underline v1.2.1
		7: Dataview v0.5.56
		8: Google Calendar v1.9.19
		9: File Explorer Note Count v1.2.1
		10: Better Word Count v0.9.6
		11: Excel to Markdown Table v0.4.0
		12: Webhook Plugin v0.0.6
		13: Advanced Tables v0.19.0
		14: Markdown Furigana v1.3.0
		15: Advanced Mobile Toolbar v1.8.0
		16: Style Settings v1.0.6
		17: Minimal Theme Settings v6.3.2

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Comment by kepano** [comment]
Do you have line numbers turned on?

**Comment by adamhl8** [comment]
Oh, interesting. Yeah turning line numbers off fixes it.

**Comment by kepano** [comment]
Ok I was able to reproduce and it will be fixed in the next update

**Comment by adamhl8** [comment]
Thanks!

**Comment by kepano** [comment]
This should be fixed in 7.1
